keyboards/AT101_Blackheart/keymaps/default/keymap.c

```diff
@@ -0,0 +1,65 @@
+#include "AT101_Blackheart.h"
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	KEYMAP(
```

Comment:
Please change the `KEYMAP`s to `LAYOUT`.

Reply:
Fixed.